src/command_modules/azure-cli-role/azure/cli/command_modules/role/generated.py

```diff
@@ -20,10 +20,10 @@
                      _auth_client_factory, _graph_client_factory)
 
 factory = lambda _: _auth_client_factory().role_definitions
-cli_command('role list', list_role_definitions)
-cli_command('role delete', delete_role_definition)
-cli_command('role create', create_role_definition)
-cli_generic_update_command('role update',
+cli_command('role definition list', list_role_definitions)
```

Comment:
Same with `resource policy`, role has 2 sub-groups `definition` and `assignment`